src/Core.hs

```diff
@@ -62,9 +65,9 @@ type ServerState = SubscriptionTree UUID WS.Connection
 newServerState :: ServerState
 newServerState = empty
 
-newCore :: IO Core
-newCore = do
-  tvalue <- newTVarIO Null
```

Comment:
This PR changes the initial value from `null` to `{}`. Not sure if that was intentional. I don’t have a strong opinion on which to take, both are sensible.

Reply:
I prefer `{}`, since the typical use case is to have one big dictionaries with values per-user or per-project etc.

Reply:
:+1: makes sense.